SubmitReview/ReadMe.md

```diff
@@ -0,0 +1,50 @@
+# SubmitReview
+
+Move fixed Jira tickets from one state to another using the `Request Changes` transitions.
```

Comment:
Please add a paragraph saying that by default, it is expected to move all the relevant tickets mentioned in the title.